tests/scripts/all.sh

```diff
@@ -2498,10 +2498,16 @@ component_test_psa_crypto_config_accel_all_curves_except_p192 () {
     # Configure and build the main libraries
     # ---------------------------------------
 
-    # start with default + driver support + USE_PSA_CRYPTO
-    scripts/config.py set MBEDTLS_PSA_CRYPTO_DRIVERS
+    # full config + USE_PSA_CRYPTO
```

Comment:
That would be "full config (includes USE_PSA_CRYPTO)" I think. (Or perhaps "includes USE_PSA, TLS 1.3 and driver support" even.)